silk/device/netns_base.py

```diff
@@ -190,6 +190,14 @@ def add_route(self, dest, dest_subnet_length, via_addr, interface_name):
                                                            via_addr, interface_name)
         self.make_netns_call_async(command, "", 1, None)
 
+    def log_debug(self, log_line: str):
```

Comment:
Should we create instance of class StandaloneNetworkNamespace(netns_name) which implements all logs ?

Reply:
Right, this makes the most sense.